src/utils/app/clean.ts

```diff
@@ -50,6 +50,7 @@ export const cleanConversationHistory = (history: any[]): Conversation[] => {
         );
 
         const cleanConversation: Conversation = {
+          ...conversation,
```

Comment:
When doing so your conversation will have correct type every time when you will change the model, but actually it's not true - you previous exports can have different formats and you will break import